scripts/updateDocs.sh

```diff
@@ -0,0 +1,42 @@
+set -eu
+
+GIT_ROOT="$(git rev-parse --show-toplevel)"
+cd $GIT_ROOT
+
+# Determine the version
+BRANCH="$(git branch | grep \* | cut -d ' ' -f2)"
```

Comment:
You are using pipes here. (`set -o pipefail` needs to be somewhere). But git allows all this sort of stuff by providing the `git rev-parse` command. Which was made for these sort of scripts. 
`git rev-parse --abbrev-ref HEAD` should do the same thing.